.cbmc-batch/source/utils.c

```diff
@@ -64,6 +64,18 @@ void assert_array_list_equivalence(
     }
 }
 
+void assert_byte_buf_equivalence(
+    const struct aws_byte_buf *const lhs,
+    const struct aws_byte_buf *const rhs,
+    const struct store_byte_from_buffer *const rhs_byte) {
+    assert(lhs->len == rhs->len);
```

Comment:
Don't you need to check that `lhs` and `rhs` are not null before you dereference `len`?

Reply:
Yes, thank you!